sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/LocalEncryptionAlgorithm.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+/**
+ * Abstract base class for all encryption implementation.
+ *
+ */
+abstract class LocalEncryptionAlgorithm extends Algorithm {
+
+    /**
+     * Constructor.
```

Comment:
Since this is package-private, you can drop the javadoc or make it more descriptive.

Reply:
removed.